include/integration/simple_MC_integration.hpp

```diff
@@ -179,7 +201,8 @@ template
     typename NT = NT,
     typename Functor
 >
-NT simple_mc_integrate (Functor Fx, 
+NT simple_mc_integrate (Functor Fx,
+                        RNG &rng,
```

Comment:
I think it is more intuitive to add `rng` parameter after `dim` since `dim` plays the role of polytope `P` of `simple_mc_polytope_integrate`.